src/Commands/Builder.php

```diff
@@ -144,6 +144,35 @@ public function where($property, $value = null, $operator = '=', $conjunction =
     }
 
     /**
+     * Set the type of the target in the current Command Bag
+     * @param $type
+     * @return $this
+     */
+    public function type($type)
```

Comment:
I haven't updated `Builder` to match the new `BaseBuilder` until `BaseBuilder` was agreed upon. `type` will add a where clause.